api/src/main/java/org/openmrs/api/db/DelegatingFullTextSession.java

```diff
@@ -56,82 +58,86 @@ public FullTextQuery createFullTextQuery(Query luceneQuery, Class<?>... entities
 		if (entities.length > 1) {
 			throw new DAOException("Can't create FullTextQuery for multiple persistent classes");
 		}
-		
+
 		if (log.isDebugEnabled()) {
-			log.debug("Creating new FullTextQuery instance");
+			log.debug("Creating new {} instance", FullTextQuery.class.getSimpleName());
```

Comment:
`log.debug("Creating new FullTextQuery instance");` does not contain a variable it is just logging the string `"Creating new FullTextQuery instance"`. So we do not need to do

`log.debug("Creating new {} instance", FullTextQuery.class.getSimpleName());`

You can simply remove the if statement `if (log.isDebugEnabled())`.